src/state/apis/zapper/validators.ts

```diff
@@ -851,6 +855,42 @@ const V2AppResponse = z.object({
   groups: z.array(V2AppGroupResponse),
 })
 
+export const parseToNftItem = (zapperItem: V2NftUserItem, chainId: ChainId): NftItem => {
+  const {
+    token,
+    token: { lastSaleEth, medias, tokenId },
+    token: { collection },
+  } = zapperItem
+
+  const collectionItem: NftCollectionItem = {
+    id: collection?.address
+      ? toAssetId({
+          assetReference: collection.address,
+          assetNamespace: collection.nftStandard as AssetNamespace,
+          chainId,
+        })
+      : '',
+    chainId,
+    description: null, // Not supported by the /v2/nft/user/tokens endpoint
```

Comment:
think this will be better
```suggestion
    description: '', // Not supported by the /v2/nft/user/tokens endpoint
```

Reply:
Ehh why not

Reply:
b7215dee6b